tests/pkg_emlearn/Makefile

```diff
@@ -0,0 +1,22 @@
+include ../Makefile.tests_common
+
+USEPKG += emlearn
+
+BLOBS += digit
+
+BUILDDEPS += sonar.h
+
+# Disable hardware FPU for Cortex-M4/7 to prevent crash on these platforms
+DISABLE_MODULE += cortexm_fpu
```

Comment:
Is this really needed?
Before we've seen crashes on `samr21-xpro` which is a cm0+ and doesn't have a FPU.

I've tried this now without the `DISABLE_MODULE += cortexm_fpu` on both  `samr21-xpro` and `same54-xpro` (cm4f) and it worked just fine.